features/step_definitions/user_steps.rb

```diff
@@ -138,9 +138,12 @@
   user.save
 end
 
+Given /^I start a new session$/ do
+  page.driver.reset!
+end
+
 Given /^I am logged out$/ do
```

Comment:
Keep both logging out steps for now is fine. Can you add a TODO on one of them saying it should be eventually removed?